x-pack/plugins/observability/public/pages/alerts/containers/alerts_page/styles.scss

```diff
@@ -4,7 +4,7 @@ $fullscreenFlyoutTop: 72px;
     top: $fullscreenFlyoutTop;
   }
 
-  .euiFlyout,
+  .euiFlyout.oblt__flyout,
   .euiCollapsibleNav {
```

Comment:
It might make sense to scope `euiCollapsibleNav` to `oblt__flyout` as well, wdyt?

Reply:
I thought so as well but I could not find any instance of EuiCollapsibleNav being used in Observability  so I left it alone.

Reply:
Okay I found that EuiCollapsible is the left sidebar collapsible Menu and this CSS applies to Collapsible menu only and only when Data grid is full screen. But when datagrid is full screen, there is no way to open the collapsible menu. See below screenshot. 

So, I guess it will be okay, we do not touch it. because I delete `.euiCollapsibleNav` from here, not sure what will break.

![image](https://user-images.githubusercontent.com/7485038/183615452-2dfb372c-cc6d-4ca7-9411-73a04466147b.png)